src/mlpack/tests/facilities_test.cpp

```diff
@@ -0,0 +1,61 @@
+/**
+ * @file facilities_test.cpp
```

Comment:
I'd suggest renaming this accordingly to match whatever you choose to rename `facilities.hpp` to. :+1:

Reply:
yeah sure.